css/style.css

```diff
@@ -0,0 +1,552 @@
+
+/* Variables */
+
+:root {
+  --color-primary: #575757;
+  --color-dark: #272a31;
+  --color-accent: #ec5242;
+  --color-negative: #d3d3d3;
+  --color-white: #fff;
+  --ff-text: 'Lato', sans-serif;
+  --ff-heading: 'Biryani', sans-serif;
+  --ff-title: 'Archivo Black', sans-serif;
+  --max-width: 1440px;
+}
+
+li {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+* {
+  font-family: var(--ff-text);
+  margin: 0;
+  padding: 0;
+}
+
+/* NAV BAR */
+
+.top-bar {
+  background-color: var(--color-primary);
+  height: 50px;
+}
+
+.top-bar ul {
+  display: flex;
+  justify-content: flex-end;
+  gap: 40px;
+  max-width: var(--max-width);
+  margin: 0 auto;
+  padding-top: 15px;
+}
+
+.top-bar a {
+  color: var(--color-white);
+  text-decoration: none;
+  font-weight: 700;
+  font-size: 16px;
+}
+
+/* nav bar */
+
+nav {
+  max-width: var(--max-width);
+  margin: 0 auto;
+}
+
+#nav-main {
+  display: flex;
+  background-color: var(--color-white);
+  height: 100px;
+  align-items: center;
+}
+
+#nav-main #logo-img {
+  max-width: 200px;
+}
+
+#menu,
+#equis {
+  display: none;
+}
+
+#nav-main .menu-links {
+  gap: 100px;
+  display: flex;
+  justify-content: flex-end;
+}
+
+/* LI PARTNERS SECTION */
+.partner-logos li {
+  /* list-style: none; */
+  font-size: 10rem;
+  text-align: center;
+  margin-left: 90px;
+}
+
+.partner-logos li:hover {
+  color: var(--color-accent);
+}
+
+/* END OF PARTNERS SECTION */
+
+#navbar-links {
+  text-align: right;
+  width: 100%;
+}
+
+#navbar-links a {
+  padding: 10px;
+}
+
+#nav-main a {
+  color: var(--color-primary);
+  font-weight: 200;
+  font-weight: 700;
+  font-size: 20px;
+  text-decoration: none;
+}
+
+#navbar-links a:hover {
+  color: var(--color-accent);
+  font-weight: bolder;
+}
+
+#navbar-links a:active {
+  outline: 2px solid #ec5242;
+  background-color: #ec5242;
+  color: var(--color-white);
+}
+
+/* main section */
+#main-section {
+  height: 88vh;
+  background-image: url(../images-capstone/main_background.png);
+}
+
+#main-section-container {
+  max-width: var(--max-width);
+
+  /* outline: 1px solid red; */
+  margin: 0 auto;
+  height: 100%;
+}
+
+#main_section_containter_two {
+  max-width: 90%;
+  height: 100%;
+  padding: 150px 0 150px 0;
+  line-height: 1;
+}
+
+.greetings {
+  color: var(--color-accent);
+  font-weight: bolder;
+  font-size: 70px;
+  font-family: var(--ff-text);
+}
+
+h1 {
+  font-family: var(--ff-title);
+  font-weight: 900;
+  font-size: 7em;
+}
+
+#main-title {
+  background-image: url(../images-capstone/main-title-background.jpg);
+  color: var(--color-white);
+  -webkit-text-fill-color: rgba(236, 82, 66, 0.6);
+  -webkit-background-clip: text;
+  background-size: 100%;
+  background-repeat: no-repeat;
+  padding-top: 35px;
+
+  /* animation-name: kenburns;
+  animation-timing-function: ease-in-out;
+  animation-duration: 20s;
+  animation-iteration-count: infinite; */
```

Comment:
- [ ] I would recommend removing commented code here and in all places to make the codebase cleaner.